tests/test_display.py

```diff
@@ -70,6 +71,20 @@ def setUp(self) -> None:
         self.font_scale = 3.0
         self.thickness = 2
 
+    @patch('sense.display.put_text')
+    def test_display_below_threshold(self, mock_put_text):
+        test_display = base_display.DisplayClassnameOverlay(thresholds={'Dabbing': 0.5})
+        test_display.display(self.img, {'sorted_predictions': [['Dabbing', 0.4]]})
+
+        mock_put_text.assert_not_called()
+
+    @patch('sense.display.put_text')
+    def test_display_class_not_in_threshold(self, mock_put_text):
+        test_display = base_display.DisplayClassnameOverlay(thresholds={'Dabbing': 0.5})
+        test_display.display(self.img, {'sorted_predictions': [['Nodding', 0.4]]})
```

Comment:
To really make clear this is not a low threshold thing
```suggestion
        test_display.display(self.img, {'sorted_predictions': [['Nodding', 1.0]]})
```